src/baldr/pathlocation.cc

```diff
@@ -39,18 +39,7 @@ namespace baldr{
     boost::property_tree::ptree correlated;
```

Comment:
it looks like `PathLocation::ToPtree` is never used anywhere (anymore, its from before we had rapidjson) so we can delete it and your new function `PathLocation::PathEdgeToPtree` as well